examples/adapters/run_tf_adapter_bert.py

```diff
@@ -0,0 +1,188 @@
+import modeling_tf_adapter_bert
```

Comment:
This file is a script to run glue right? I feel it would be more aptly named `run_tf_glue_adapter_bert` or something along those lines. What do you think?

Reply:
Great!